draft-ietf-ppm-dap.md

```diff
@@ -613,7 +622,7 @@ standard tokens for use in the "type" field (within the DAP URN namespace
 | batchQueriedMultipleTimes  | A batch was queried with multiple distinct aggregation parameters. |
 | batchMismatch              | Aggregators disagree on the report shares that were aggregated in a batch. |
 | unauthorizedRequest        | Authentication of an HTTP request failed (see {{request-authentication}}). |
-| stepMismatch               | The Aggregators disagree on the current step of the DAP aggregation protocol. |
+| stepMismatch               | The Aggregators disagree on the current step of the DAP aggregation sub-protocol. |
```

Comment:
Orthongonal nit: back when we got a review from HTTPDIR, we got a recommendation to use the term "interaction" or "sub-interaction" instead of "sub-protocol". Mostly this was cleaned up in #450, but I appear to have missed this one.